opportunity/README.md

```diff
@@ -2,7 +2,22 @@
 
 - **Conformance URI:** <https://stapi.example.com/v0.1.0/opportunities>
 
-The STAPI Opportunity describes a single business unit available for ordering.
+An Opportunity in STAPI is an abstract, and often terse, representation
+of data that will be delivered in the future. Because of the uncertainty
+involved an Opportunity may contain a range of values rather than 
+discrete known values. The amount of metadata fields contained within an 
+Opportunity will vary depending on the user requirements and what is known
+by the provider at the time.
+
+**An Opportunity is the minimum amount of metadata known about the data
+that will be delivered**.
+
+The optional `POST /products/{productId}/opportunities` endpoint provides
+additional functionality on top of the core STAPI spec. It allows
+a user to search and browse additional information about what will be 
+delivered after an order is made. The `opportunities` endpoint is specific
+for a product and would be used before a user places an order with a call to
```

Comment:
```suggestion
for a product and could be used before a user places an order with a call to
```

Thinking about our CSDA use-case, we may actually use opportunities in our backend processes; leading me to suggest that the matter of when to use (before or after an order), could be more of an implementation choice (alongside of the choice of whether to use opportunities at all).